static/js/emoji.js

```diff
@@ -261,22 +261,37 @@ export function get_emoji_details_for_rendering(opts) {
     };
 }
 
+function build_default_emoji_aliases({names, get_emoji_codepoint}) {
+    // Please keep this as a pure function so that we can
+    // eventually share this code with the mobile codebase.
```

Comment:
We probably don't need these comments, in that I don't think we're likely to let someone go back to the ugly globals approach in any case :)

Reply:
All that being said, I don't this to be a huge blocker, so if you just want to snip them or edit them yourself before merging, please do that without another review cycle.

Reply:
I think we can leave them for now; we can drop them when we move the pure functions in a proper shared module down the line.